File: CH1-input-process-output/EX5-distance-traveled/source-code/distanceTraveled.js
const prompt = require("prompt-sync")();

var distance;
var time;
var speed = 60;

// Input
time = Number(prompt("Enter a time : "));

// Process
distance = speed * time;

// Output
console.log(
  "Distance the car travel in " + time + " hours is : " + distance + "miles"
);

File: CH1-input-process-output/EX5-distance-traveled/source-code/distanceTraveled.cpp
#include <iostream>
using namespace std;

int main()
{
    int distance;
    int speed = 60;
    int time;

    // Input
    cout << "Enter a time : ";
    cin >> time;

    // Process
    distance = speed * time;

    // Output
    cout << "Distance the car travel in " << time << " hours is : " << distance << " Miles";

    return 0;
}